app/internal_packages/list-unsubscribe/lib/main.tsx

```diff
@@ -60,18 +57,22 @@ const regexps = [
   /notisinställningar/gi,
 ];
 
+const _throwawayCache: Record<string, string> = {};
```

Comment:
It might be nice to use an `LRU Cache` here (I think we have an lru-cache module already) so that this object doesn't grow indefinitely, just on the off chance that someone views 100k messages without restarting Mailspring 😅 I think it's a small risk though, maybe we change it if anyone ever notices!